### ISSUE TITLE
[Idea] Add other waveforms than sine

### ISSUE THREAD

**Issue by probonopd** [probonopd/MiniDexed#87]
DX7 and Dexed only have sine waves. The TX81Z can generate 7 additional shapes:

![image](https://user-images.githubusercontent.com/2480569/163018138-489a694a-4e12-475d-9ff3-d357d0bc7d08.png)

WAV versions: https://www.audiobombs.com/items/1083/joel%27s-synth-waveforms

The Korg Opsix is doing a similar thing: https://www.youtube.com/watch?v=yS693SgnBws

How hard would it be to implement this in Synth_Dexed (and MiniDexed) @dcoredump?

Reference:
[Reddit](https://www.reddit.com/r/synthdiy/comments/syup5a/comment/hy1fynf/?utm_source=share&utm_medium=web2x&context=3)

**Comment by dcoredump** [comment]
> How hard would it be to implement this in Synth_Dexed (and MiniDexed) 

Good question 😁

Dexed has fixed arrays (lookup tables), so exchanging only the sine table looks like the solution, BUT there are also log() LUTs, so I have currently no idea if they are related to the sine oscillators.

Maybe something to try for someone else because my list for fixes and extensions is long 😀

**Comment by probonopd** [comment]
Yes. Let's leave it as an idea, maybe someone comes along who whould like to have a go at it.
